pkg/util/log/types/types.go

```diff
@@ -101,3 +106,59 @@ const (
 func (level LogLevel) String() string {
 	return seelog.LogLevel(level).String()
 }
+
+const (
+	slogTraceLvl    slog.Level = slog.LevelDebug - 4
+	slogDebugLvl    slog.Level = slog.LevelDebug
+	slogInfoLvl     slog.Level = slog.LevelInfo
+	slogWarnLvl     slog.Level = slog.LevelWarn
+	slogErrorLvl    slog.Level = slog.LevelError
+	slogCriticalLvl slog.Level = slog.LevelError + 4
+	slogOff         slog.Level = slog.LevelError + 8
+)
+
+// ToSlogLevel converts a LogLevel to a slog.Level
+func ToSlogLevel(level LogLevel) slog.Level {
+	switch level {
+	case TraceLvl:
+		return slogTraceLvl
+	case DebugLvl:
+		return slogDebugLvl
+	case InfoLvl:
+		return slogInfoLvl
+	case WarnLvl:
+		return slogWarnLvl
+	case ErrorLvl:
+		return slogErrorLvl
+	case CriticalLvl:
+		return slogCriticalLvl
+	case Off:
+		return slogOff
+	default:
+		// unreachable
+		panic(fmt.Sprintf("unknown log level: %d", level))
+	}
+}
+
+// FromSlogLevel converts a slog.Level to a LogLevel
+func FromSlogLevel(level slog.Level) LogLevel {
```

Comment:
😭 the type mismatch between these is so unfortunate

```
seelog.LogLevel uint8
slog.Level int
```

Reply:
Yeah and they don't even use the same default value (`slog.Level`'s 0 value is `Info`, while `seelog` is `Trace`), and they don't have the same diff between consecutive values
For now I'm using `FromSlogLevel` and `ToSlogLevel` so that we can stick to using `seelog` `LogLevel` as the "main" level), but eventually they'll be removed and we'll only use the `slog` values

In terms of conversion we could do something like the following if we needed something more efficient, but I thought a switch case would be more explicit
```go
func FromSlogLevel(level slog.Level) LogLevel {
	return LogLevel(level/4+2)
}